llama_stack/core/distribution.py

```diff
@@ -264,9 +265,16 @@ def get_external_providers_from_module(
                         config_class="",
                     )
                 provider_type = provider.provider_type
-                # in the case we are building we CANNOT import this module of course because it has not been installed.
-                # return a partially filled out spec that the build script will populate.
-                registry[Api(provider_api)][provider_type] = spec
+                if isinstance(spec, list):
```

Comment:
where is the corresponding type definition change which changes the type into a `ProviderSpec | list[ProviderSpec]`?